<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgFood</class>
 <widget class="QWidget" name="WdgFood">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="WdgDays">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QToolButton" name="tbAdd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>1</y>
        <width>38</width>
        <height>38</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="foodcl.qrc">
        <normaloff>:/resources/plus.png</normaloff>:/resources/plus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="tbDel">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>1</y>
        <width>38</width>
        <height>38</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="foodcl.qrc">
        <normaloff>:/resources/minus.png</normaloff>:/resources/minus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="tbAddGroup">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>1</y>
        <width>38</width>
        <height>38</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="foodcl.qrc">
        <normaloff>:/resources/addgrp.png</normaloff>:/resources/addgrp.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="tbMove">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>1</y>
        <width>38</width>
        <height>38</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="foodcl.qrc">
        <normaloff>:/resources/prmove.png</normaloff>:/resources/prmove.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>40</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Search:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leFind">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="tbNext">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>1</y>
        <width>38</width>
        <height>38</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="foodcl.qrc">
        <normaloff>:/resources/next.png</normaloff>:/resources/next.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="tbEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>1</y>
        <width>38</width>
        <height>38</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="foodcl.qrc">
        <normaloff>:/resources/pen.png</normaloff>:/resources/pen.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="lbFound">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>12</y>
        <width>20</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbFound_2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>12</y>
        <width>14</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>of</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbFoundCount">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>12</y>
        <width>30</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QToolButton" name="tbPrev">
      <property name="geometry">
       <rect>
        <x>472</x>
        <y>1</y>
        <width>38</width>
        <height>38</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="foodcl.qrc">
        <normaloff>:/resources/prev.png</normaloff>:/resources/prev.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="foodcl.qrc"/>
 </resources>
 <connections/>
</ui>
